dataland-frontend/tests/e2e/fixtures/frameworks/sfdr/SfdrDataFixtures.ts

```diff
@@ -1,198 +1,186 @@
-import { DEFAULT_PROBABILITY, Generator } from "@e2e/utils/FakeFixtureUtils";
-import {
-  generateCurrencyValue,
-  generateFloat,
-  generateInt,
-  generatePercentageValue,
-} from "@e2e/fixtures/common/NumberFixtures";
-import { type SfdrData, type SfdrHighImpactClimateSectorEnergyConsumption } from "@clients/backend";
-import { generateFiscalYearDeviation } from "@e2e/fixtures/common/FiscalYearDeviationFixtures";
-import { generateYesNo } from "@e2e/fixtures/common/YesNoFixtures";
-import { generateFutureDate } from "@e2e/fixtures/common/DateFixtures";
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+import { DEFAULT_PROBABILITY } from "@e2e/utils/FakeFixtureUtils";
 import { type FixtureData } from "@sharedUtils/Fixtures";
-import { generateFixtureDataset, pickSubsetOfElements } from "@e2e/fixtures/FixtureUtils";
-import { HighImpactClimateSector } from "@/api-models/HighImpactClimateSector";
+import { type SfdrData } from "@clients/backend";
+import { SfdrGenerator } from "@e2e/fixtures/frameworks/sfdr/SfdrGenerator";
+import { pickOneElement, generateFixtureDataset } from "@e2e/fixtures/FixtureUtils";
+import { SfdrGeneralGeneralFiscalYearDeviationOptions } from "@clients/backend";
```

Comment:
Hier gibt es ein Duplikat der importe mit "@clients/backend" das sollte nur eine Zeile sein statt wie jetzt zwei

Reply:
Das müsste an dieser Zeile
`import { type ${frameworkBaseName}Data } from "@clients/backend";`
in der Datei `DataFixtures.ts.ftl` liegen. Wenn weitere Importe über die Liste der Importe für @clients/backend hinzukommen entsteht das Duplikat.

Reply:
Klären wir in der EU Taxonomy Story